doc/compiling/macos.md

```diff
@@ -1,5 +1,7 @@
 # Compiling on MacOS
 
+If you're just creating mods or games with Luanti, you do not need to compile Luanti. Instead, follow Ruben Wardy's [Luanti Modding Book](https://rubenwardy.gitlab.io/minetest_modding_book) to get started modding. Compiling Luanti is only required if you plan to modify the Luanti engine itself.
```

Comment:
IMO we don't need this notice pasted over every instructions, but this PR isn't the right place anyway.

Reply:
Sure, I'll remove it from this PR